sensing/pointcloud_preprocessor/docs/ring-outlier-filter.md

```diff
@@ -22,12 +22,12 @@ This implementation inherits `pointcloud_preprocessor::Filter` class, please ref
 
 ### Core Parameters
 
-| Name                      | Type    | Default Value | Description |
-| ------------------------- | ------- | ------------- | ----------- |
-| `distance_ratio`          | double  | 1.03          |             |
-| `object_length_threshold` | double  | 0.1           |             |
-| `num_points_threshold`    | int     | 4             |             |
-| `max_rings_num`           | uint_16 | 128           |             |
+| Name                      | Type    | Default Value | Description                  |
+| ------------------------- | ------- | ------------- | ---------------------------- |
+| `distance_ratio`          | double  | 1.03          |                              |
+| `object_length_threshold` | double  | 0.1           |                              |
+| `num_points_threshold`    | int     | 4             |                              |
+| `max_ring_index`          | uint_16 | 128           | `128` corresponds to vls128. |
```

Comment:
For index, I am not sure if it is 127 or 128 when using vls128. max_rings_num would be easier to understand if it is 128.

Reply:
related comment: https://github.com/autowarefoundation/autoware.universe/pull/4981#discussion_r1325316752
cc @taikitanaka3

Reply:
@yukkysaito 

> In a project using Autoware, max_rings_num was set to 128 (i.e. the range of ring index was set to be 0~127), but actually the maximum value of ring index was 128.
To prevent this kind of thing, the parameter name should be changed to be less misleading.

Based on this explanation, wouldn't it be more appropriate to use 'max_rings_index' rather than 'max_rings_num'?

Reply:
@tkimura4 If it weren't for the description, I would think that vls128 would be correct to specify 127 based on the name max_ring_index.